Documentation/gettingstarted/k8s-install-gke.rst

```diff
@@ -101,6 +101,6 @@ to the cluster. The NodeInit DaemonSet will perform the following actions:
 
 .. include:: k8s-install-restart-pods.rst
 .. include:: k8s-install-gke-validate.rst
-.. include:: namespace-cilium.rst
```

Comment:
For other reviewers: we can't delete this file as we still need it for Azure CNI.